boeing_voucher/pdf_parsing.py

```diff
@@ -0,0 +1,153 @@
+"""PDF Parsing functions for Boeing Vouchers"""
+
+import logging
+import pdftotext
+
+log = logging.getLogger()
+
+
+def read_pdf_domestic(pdf):
+    """
+    Process domestic Boeing vouchers and return parsed values
+    """
+    row_values = {"Date": "", "Employee Name": "", "BEMSID": ""}
+
+    column_values = {
+        "Voucher ID": "",
+        "Course Name": "",
+        "Credits/CEUs": "",
+        "Course Dates": "",
+        "Pre-Authorized Amount": "",
+    }
```

Comment:
Not sure if this is a valid concern or not, but maybe someone looking at the code could figure out how to generate their own PDF with the required values to get a discount.  One way to obfuscate them a bit would be to put the required fields in `.env`/settings variables. something like this:

```python
row_values = {settings.VOUCHER_DATE_KEY: "",  settings.VOUCHER_EMPLOYEE_KEY: "", settings.VOUCHER_BEMS_KEY: ""}
column_values = {
    settings.VOUCHER_KEY,
    settings.VOUCHER_COURSE_KEY,
    settings.VOUCHER_CREDITS_KEY,
    settings.VOUCHER_COURSE_DATES_KEY,
    settings.VOUCHER_AMOUNT_KEY
}
```

and then for the unit tests set these to some bogus values and use those bogus keys in the test pdf's.

Reply:
@Ferdi How much of a concern is this?

Reply:
I spoke with Ferdi and he thinks we should go ahead and make this change. Let me know if you need new sample PDFs.

Reply:
I have been making do with LibreOffice Draw for now. I'll let you know if it isn't sufficient for this task.

Reply:
@mbertrand @pdpinch Is this what you were looking for? Should I do the other strings in the international pdf as well or is this sufficient?